charts/cloudzero-agent/values.yaml

```diff
@@ -47,7 +47,7 @@ prometheusConfig:
 kube-state-metrics:
   enabled: false
   extraArgs:
-    - --metric-labels-allowlist=namespaces=[*],pods=[*],deployments=[app.kubernetes.io/*,k8s.*]
+    - --metric-labels-allowlist=pods=[app.kubernetes.io/instance]
```

Comment:
why not `app.kubernetes.io/name`, or or `app.kubernetes.io/component`?

Reply:
switched to `app.kubernetes.io/component`